### ISSUE TITLE
Invisible characters in LaTeX

### ISSUE THREAD

**Issue by martonbalazs** [xournalpp/xournalpp#4018]
(Please complete the following information, and then delete this line)

**Affects versions :**
 - OS: [e.g. Linux (which distro), Mac, Windows]

Arch Linux

 - (Linux only) Desktop environment: [e.g. Wayland, Gnome-Wayland, XWayland, X11]

i3 WM

 - Which version of libgtk do you use
     - can be obtained by running `xournalpp --version` or in the **About** window

3.24.33 

 - Version of Xournal++: [e.g. 1.0.16, specific commit, specific branch]
     - can be obtained by running `xournalpp --version` or in the **About** window

1.1.1   

 - Installation method: [e.g. PPA, Flatpak, official release, etc.]

Tried both xournalpp from Community and xournalpp-git from AUR, same result

**Describe the bug**

LaTeX produces invisible characters both in the popup window and in the document. The .tex, .pdf and .log files in /tmp all look normal.

**To Reproduce**

Try pretty much anything LaTeX in Xournalpp.

**Expected behavior**
I was hoping to see it all, not only a few bits of the LaTeX.

**Screenshots of Problem**
See the issue, the pdf file from /tmp (looks ok) and the end of the log file (also looks ok).

**Additional context**
--
![xournalpp_latex_invisible_220509](https://user-images.githubusercontent.com/8852115/167460639-980efecb-a697-4696-85d1-0f0f9d506aae.jpg)

**Comment by tattsan** [comment]
I've run into a similar bug on `evince`. It was resolved if I revert `poppler-*` packages to ver 22.02.0 .

**Comment by Technius** [comment]
@martonbalazs Could you confirm whether it's poppler's issue? What PDF viewer is used at the bottom left?

**Comment by martonbalazs** [comment]
@Technius Yes, I can confirm that things work well with
poppler-22.03.0-1       poppler-glib-22.03.0-1        poppler-qt5-22.03.0-1
and fail as in my first post with
poppler-22.05.0-1       poppler-glib-22.05.0-1        poppler-qt5-22.05.0-1

Thanks Both for the hint. Would it help for me to file a bug report to them or is it better done by someone professional?

The pdf viewer is Zathura but I don't think it's related to the bug. (Try it, I love Zathura! :-) )

**Comment by tattsan** [comment]
Zathura has a choice either poppler or mupdf as the rendering engine for PDF. In Arch Linux, their package names are `zathura-pdf-poppler` and `zathura-pdf-mupdf`.

**Comment by martonbalazs** [comment]
@tattsan Ok, it was zathura-pdf-mupdf 0.3.8-1. I now removed it and installed zathura-pdf-poppler 0.3.0-1 out of curiosity; both seem to work fine.

**Comment by tattsan** [comment]
For `evince + poppler-* 22.05.0-1`, the bug will appear as follows:
- Display a PDF generated by TeX with evince. At this time, it is displayed normally.
- Modify the TeX source and generate a modified PDF while displaying the  PDF. Then the display becomes abnormal.

**Comment by tattsan** [comment]
I've confirmed that this bug also occurs on `zathura-pdf-poppler + poppler-* 22.05.0-1` and on `xournalpp + poppler-* 22.05.0-1`.

**Comment by tattsan** [comment]
I've reported this issue upstream.
https://gitlab.freedesktop.org/poppler/poppler/-/issues/1248

**Comment by martonbalazs** [comment]
Indeed, both zathura-pdf-poppler and evince with poppler-22.05.0-1 has the same problem as xournalpp in my original post when changing the pdf underneath: the changed bits do not appear normally in the viewer anymore. (Sorry for not having connected the dots before and not going this far, writing zathura-pdf-poppler worked ok: it didn't). The problem with both viewers is solved by downgrading to poppler-22.03.0-1.

**Comment by tattsan** [comment]
On Arch Linux, poppler-* 22.05.0-2  will fix this bug. Now it's in testing.
